playbooks/post-run.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# curl -k https://dashboard.zuul.ansible.com/api/tenant/ansible/key/github.com/ansible/ansible-navigator.pub -o ansible-navigator.pub
+# zuul-client encrypt --public-key ansible-navigator.pub
+- secret:
+    name: codecov
+    data:
+      token: !encrypted/pkcs1-oaep
+        - qald8OkWhQOoTsF/u47Vlo2raMblcQyDWwphXAiVJe+FeNuWAww5pFvmHFi6L0YoPipF6
+          PxEv0ynn8H5Dnf4/I+mGOF0RNg1aU8k7QnZkeqhvKblPEPbX8qbIjoDd/RituMPHNsrBZ
+          oCRmkuazZ+/UJSlu2ZTd7L/W9IWP4+q8bMgs7/0mn2yG5OxyKCm7M4nPbudSch6jM0bmA
+          G7UblqWL2Sl7cXSWytp/GNsFBprjpZUnzEaJHSAjT29hTa/zXT2sEarsR0eThFCwlPj/A
+          ckPguAm9E5XJuyry+KpEOPS+aZy+rXK51+i7V0spkHGHM9XLyyZRcEToJmsRzyyYjCzII
+          9f5iYphFohQBHoA62BBzAnifxQ2MH9nHo1+Kkc6wQRkYK4UqTC0J83mhA/bT8fgdIoopz
+          mFgD3m4j2DnVavvorOZ9S/59FswqqJbtnOj0UlGz2OwQGrcUKWcphtL7Wx/dZOoPAIduh
+          9ClGKWBuYNakkiwAqPszlCTs39iuev5Af32O8S3VLf+aYAdAGIvYhSPsm3i46dNotEzzI
+          ZmCJ+uv5eDNAhP5ZVPzDkSrt3Ka0GtVvYolLQu9WwZClTnDX1ILwLsUe3IiHdxpYn5YrV
+          UT5ImZ8kwOGFnpvJyJsJy747dJfLiqLSxIgTjL3b4hpHJ/vHhvgaZyGrKeNHHs=
+
+- hosts: all:!appliance
+  tasks:
+    - name: Coverage report
+      environment:
+        CODECOV_TOKEN: "{{ codecov.token }}"
+      command:
+        set -ex
+        pip install 'coverage[toml]'
+        coverage combine .tox/.coverage.*
```

Comment:
This shouldn't be necessary unless tox itself is executed in parallel mode. AFAIK only tox is.